src/eslhooks.h

```diff
@@ -90,9 +90,7 @@ namespace eslhooks
 
 		static void AdjustFormIDFileIndex(RE::TESFile* a_file, RE::FormID& a_formID)
 		{
-			if (a_file->IsLight()) {
-				logger::debug("Adjust form {:x} for file {}", a_formID, std::string(a_file->fileName));
-			}
+			logger::trace("Adjust form {:x} for file {} (isLight: {})", a_formID, std::string(a_file->fileName), a_file->IsLight());			
```

Comment:
As a todo we should probably decide on convention on our logger statements. Doesn't have to be now.

Reply:
I agree. I moved these to trace given how these can easily be 1000+ lines being logged in a few seconds